### PR TITLE
marking hebmorph-lucene.2.0-BETA1

### PR DESCRIPTION
**Pull request by EgozyN**

### CODE REVIEW

File: java/pom.xml [hunk 1]
```diff
@@ -9,7 +9,7 @@
     <groupId>com.code972.hebmorph</groupId>
     <artifactId>hebmorph-lucene</artifactId>
     <packaging>jar</packaging>
-    <version>2.0.0-SNAPSHOT</version>
+    <version>2.0-BETA1</version>
```
Comment (synhershko) [inline java/pom.xml]: 2.0.0-BETA1

File: java/pom.xml [hunk 2]
```diff
@@ -52,14 +52,17 @@
             </plugin>
             <plugin>
                 <groupId>org.apache.maven.plugins</groupId>
-                <artifactId>maven-release-plugin</artifactId>
-                <version>2.2.1</version>
+                <artifactId>maven-jar-plugin</artifactId>
```
Comment (synhershko) [inline java/pom.xml]: why this change?

Comment (EgozyN) [inline java/pom.xml]: Release plugin wasn't really used for anything. Jar plugin allows us to generate an additional file named hebmorph-lucene.jar automatically.

Comment (synhershko) [inline java/pom.xml]: mvn package already does that - and we don't need hebmorph-lucene.jar - we need the file name to be versioned
